ml-proto/host/format.mli

```diff
@@ -0,0 +1,7 @@
+open Sexpr
+
+val func_type : Types.func_type -> sexpr
+
+val expr : Kernel.expr -> sexpr
+val module_ : Kernel.module_ -> sexpr
```

Comment:
I wonder if `Format` is the right module name here; "Format" sounds like it generates formatted text output.